src/utils/getTimeoutForWorkerTask.js

```diff
@@ -23,6 +23,9 @@ const getTimeoutForWorkerTaskUncapped = (state, taskName, options) => {
     case 'DifferentialExpression': {
       return 180;
     }
+    case 'TrajectoryAnalysis': {
+      return 300;
```

Comment:
Set to be longer because the work is long

Reply:
I think it should because it takes into account the expression in each cell.

Reply:
I haven't checked the relation between the number of cells and the length of the runtime.